packages/icons/src/svgs/moonbeam-colorful.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14.1 11.8">
+<style type="text/css">
+	.st0{fill:#5FC0C1;}
```

Comment:
maybe its better to change the class name?

Reply:
class name duplicate

![image](https://github.com/ant-design/ant-design-web3/assets/18319675/af65dcf4-c187-4f1b-8f0e-e319f25e385d)

Reply:
> maybe its better to change the class name?

Thank you for pointing out the issue, but I'm wondering if there is really a need for semanticization. These icons generally don't require modification because many SVG files are converted from other formats like PNG by machines.

Reply:
> maybe its better to change the class name?

It's really difficult to give a meaningful name to a specific element of a logo unless I am the designer of that logo, haha, just kidding. But it is indeed challenging for me.

Reply:
> class name duplicate
> 
> ![image](https://private-user-images.githubusercontent.com/18319675/293636631-af65dcf4-c187-4f1b-8f0e-e319f25e385d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQxNzg5MTcsIm5iZiI6MTcwNDE3ODYxNywicGF0aCI6Ii8xODMxOTY3NS8yOTM2MzY2MzEtYWY2NWRjZjQtYzE4Ny00ZjFiLThmMGUtZTMxOWYyNWUzODVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAyVDA2NTY1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0ZmNlN2U1ODYyY2Q4MDdlMzAxMDM4OTcxY2RiMTgxNDQ4YmQ5YmQwODAwMWNkOWU5OWI4OTQzMGQwZmQ2ODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bAOVzkopwPSRFxy0eurZijosrZSVh35_9L4bpkPwvt0)

Is there any problem with having duplicate class names in different workspaces?